galpy/potential/MovingObjectPotential.py

```diff
@@ -1,36 +1,28 @@
 ###############################################################################
 #   MovingObjectPotential.py: class that implements the potential coming from
 #                             a moving object
-#                                                           GM
-#                              phi(R,z) = -  ---------------------------------
-#                                                        distance
 ###############################################################################
 import copy
 import numpy as nu
-from .Potential import Potential, _APY_LOADED
+import warnings
+from .Potential import Potential, _APY_LOADED, _isNonAxi, flatten, \
+    evaluatePotentials, evaluateRforces, evaluatezforces, evaluateDensities, \
+    evaluatephiforces
+from .PlummerPotential import PlummerPotential
 if _APY_LOADED:
     from astropy import units
 from .ForceSoftening import PlummerSoftening
+from galpy.util import galpyWarning
 class MovingObjectPotential(Potential):
-    """Class that implements the potential coming from a moving object
-
-    .. math::
-
-        \\Phi(R,z,\\phi,t) = -\\mathrm{amp}\\,GM\\,S(d)
-
-    where :math:`d` is the distance between :math:`(R,z,\\phi)` and the moving object at time :math:`t` and :math:`S(\\cdot)` is a softening kernel. In the case of Plummer softening, this kernel is
-
-    .. math::
-
-        S(d) = \\frac{1}{\\sqrt{d^2+\\mathrm{softening\_length}^2}}
-
-    Plummer is currently the only implemented softening.
-
     """
-    def __init__(self,orbit,amp=1.,GM=.06,
+    Class that implements the potential coming from a moving object by combining
```

Comment:
Keep the defaults from before (currently, there is no reasonable default): ``GM=0.06``, ``softening_length=0.01``; I think we should remove ``softening_model``.